docs/contributing/coding-guidelines/ros-nodes/parameters.md

```diff
@@ -1,5 +1,70 @@
 # Parameters
 
-!!! warning
+Autoware has following parameters
 
-    Under Construction
+- `p1`: reference parameter for the node package
+  - e.g., [the parameter for the `behavior_path_planner` package](https://github.com/autowarefoundation/autoware.universe/tree/main/planning/behavior_path_planner/config)
+- `p2`: reference parameter for the module launch
+  - e.g., [the parameter for the `tier4_planning_launch` package](https://github.com/autowarefoundation/autoware.universe/tree/main/launch/tier4_planning_launch/config/)
+- `p3`: [reference parameter for the `autoware_launch` package](https://github.com/autowarefoundation/autoware_launch/tree/main/autoware_launch/config)
```

Comment:
How about converting this list to a table?
Also, I feel it's better to use more descriptive names for `p1`, `p2`, and `p3`.
For example, `Node Parameter`, `Component Parameter`, and `Launch Parameter`? :thinking:

Reply:
Also, is it possible to add a figure to visually explain these types of parameters?

Reply:
Thank you for your good idea. I've fixed what I agreed.
The explanation is not short words but sentences, so I feel it better to use itemization rather than the table.